src/cloud-code/httpRequest.js

```diff
@@ -1,25 +1,36 @@
 var request = require("request"),
   Parse = require('parse/node').Parse;
 
-var encodeBody = function(body, headers = {}) {
+var encodeBody = function(options = {}) {
+  let body = options.body;
+  let headers = options.headers || {};
   if (typeof body !== 'object') {
-    return body;
+    return options;
   }
   var contentTypeKeys = Object.keys(headers).filter((key) => {
     return key.match(/content-type/i) != null;
   });
 
-  if (contentTypeKeys.length == 1) {
+  if (contentTypeKeys.length == 0) {
+    // no content type
+    try {
+      options.body = JSON.stringify(body);
+      options.headers = options.headers || {};
+      options.headers['Content-Type'] = 'application/json';
+    } catch(e) {
+      // do nothing;
+    }
+  } else if (contentTypeKeys.length == 1) {
```

Comment:
There should only be one.. not sure what to do if there's more than one... what does cURL do?